### PR TITLE
Nodes and Edges to Classes

### PR DESCRIPTION
**Pull request by jlaura**

### CODE REVIEW

File: functional_tests/test_two_image.py [hunk 1]
```diff
@@ -47,39 +45,34 @@ def test_two_image(self):
...
+        for i, node in cg.nodes_iter(data=True):
+            self.assertIn(node.nkeypoints, range(490, 511))
 
         # Step: apply Adaptive non-maximal suppression
-        for node, attributes in cg.nodes_iter(data=True):
-            attributes['keypoint_mask'] = od.adaptive_non_max_suppression(attributes['keypoints'])
-            self.assertEqual(len(attributes['keypoint_mask']), len(attributes['keypoints']))
+        for i, node in cg.nodes_iter(data=True):
+            pass
+            #node.anms()
```
Comment (kree) [inline functional_tests/test_two_image.py]: Why is the ANMS test taken out?

Comment (jlaura) [inline functional_tests/test_two_image.py]: The runtimes are >12 seconds per edge right now.  It is on my list to checkout why the runtimes are so long (how to remove the nested for loop really).

Comment (kree) [inline functional_tests/test_two_image.py]: Sounds good!

File: autocnet/graph/network.py [hunk 2]
```diff
@@ -14,6 +14,322 @@
...
+        else:
+            mask = threshold_mask
+
+        self.subpixel_offsets = pd.DataFrame(full_offsets, columns=['x_offset',
+                                                                             'y_offset',
+                                                                             'correlation'])
+        self.masks = ('subpixel', mask)
+
+    def convex_hull_coverage(self, clean_keys=[]):
+        """
```
Comment (kree) [inline autocnet/graph/network.py]: This function doesn't look finished? If intentional, could you just stick a # TODO in there?

File: autocnet/graph/network.py [hunk 3]
```diff
@@ -14,6 +14,322 @@
...
+        return self._masks
+
+    @masks.setter
+    def masks(self, v):
+        self._masks.add(v[0])
+        self._mask_arrays[v[0]] = v[1]
+
+    def get_array(self, band=1):
+        """
+        Downsample the input image file by some amount using bicubic interpolation
```
Comment (kree) [inline autocnet/graph/network.py]: Doesn't look like it's actually being downsampled? Perhaps update comment?

File: autocnet/graph/network.py [hunk 4]
```diff
@@ -507,7 +698,7 @@ def _validate_cnet(cnet):
 
         # Final validation to remove any correspondence with multiple correspondences in the same image
         merged_cnet = _validate_cnet(merged_cnet)
-
+        print(merged_cnet)
```
Comment (kree) [inline autocnet/graph/network.py]: print statement left in

File: autocnet/graph/network.py (hunk 3 above)
Comment (jlaura) [inline autocnet/graph/network.py]: Good catch.

**Comment by kree** [comment]
:+1:

**Pull request merged**